query/src/main/java/org/infinispan/query/indexmanager/ClusteredSwitchingBackend.java

```diff
@@ -146,13 +146,18 @@ public synchronized void lazyInitialize() {
    }
 
    private synchronized void applyViewChangedEvent(ViewChangedEvent e) {
-      assert e != null;
-      assert e.getNewMembers().size() > 0;
-      assert e.getNewMembers().get(0) != null;
+      List<Address> newMembers = e.getNewMembers();
       if (log.isDebugEnabled()) {
-         log.debug("Notified of new View! Members: " + e.getNewMembers());
+         log.debug("Notified of new View! Members: " + newMembers);
```

Comment:
`isDebugEnabled()` shouldn't be necessary anyway -- if the message is logged so often that the string concatenation influences performance, it should be `trace` instead of `debug`.